tests/Maker/MakeResetPasswordTest.php

```diff
@@ -81,6 +82,34 @@ public function getTestDetails(): \Generator
             }),
         ];
 
+        yield 'it_detects_symfony_cli_usage' => [$this->createMakerTest()
+            ->run(function (MakerTestRunner $runner) {
+                $this->makeUser($runner);
+
+                $output = $runner->runMaker(
+                    inputs: ['App\Entity\User', 'app_home', 'victor@symfonycasts.com', 'SymfonyCasts'],
+                    envVars: [CliOutputHelper::ENV_VERSION => '1.0.0', CliOutputHelper::ENV_BIN_NAME => 'symfony']
+                );
+
+                $this->assertStringContainsString('Success', $output);
+                $this->assertStringContainsString('symfony console make:migration', $output);
+            }),
+        ];
+
+        yield 'it_detects_symfony_cli_is_not_used' => [$this->createMakerTest()
+            ->run(function (MakerTestRunner $runner) {
+                $this->makeUser($runner);
+
+                $output = $runner->runMaker(
+                    inputs: ['App\Entity\User', 'app_home', 'victor@symfonycasts.com', 'SymfonyCasts'],
+                    envVars: []
+                );
+
+                $this->assertStringContainsString('Success', $output);
+                $this->assertStringContainsString('php bin/console make:migration', $output);
+            }),
+        ];
```

Comment:
Let's just add a test for the correct command to ONE of these makers (not all of them) just as a smoke test. It's too much added test code (and test running time) for such a minor detail.